src/utils/IdealGraphVisualizer/.gitignore

```diff
@@ -1,2 +1,6 @@
 /*/target
-.igv.log
\ No newline at end of file
+.igv.log
+/all-*.dat
+/lastModified/
+/localeVariants
+/package-attrs.dat
```

Comment:
Is that really needed? Just wondering why these files haven't been added before.

Reply:
Not sure, IGV generates them at first run on my system (after a `git clean` and clean IGV build).

Reply:
I'll remove this from the PR and make a separate JBS issue.